runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/util/common/worker/MapTaskExecutor.java

```diff
@@ -72,9 +69,7 @@ public void execute() throws Exception {
     // Save the current thread that is executing so that abort() can interrupt it, we save it before
     // starting the progress reporter thread, therefore ensuring thread safety through implicit
     // serialization of events.
-    synchronized (this) {
-      this.currentExecutorThread = Thread.currentThread();
-    }
+    // Current thread that execute() is running on; null when not running.
```

Comment:
This whole comment can be removed, it seems to be referring to logic that no longer exists (`abort` doesn't reference `currentExecutorThread`.

Reply:
Removed